commons/src/main/res/values/dimens.xml

```diff
@@ -31,6 +34,9 @@
     <dimen name="rounded_corner_radius_big">10dp</dimen>
     <dimen name="bottom_sheet_corner_radius">24dp</dimen>
     <dimen name="breadcrumbs_layout_height">48dp</dimen>
+    <dimen name="bottom_cab_layout_height">48dp</dimen>
+    <dimen name="bottom_cab_layout_max_width">300dp</dimen>
+    <dimen name="bottom_cab_layout_item_size">48dp</dimen>
```

Comment:
some of these newly added dimensions arent used even in App Launcher, I dont think they are needed, right?